mem0/memory/storage.py

```diff
@@ -139,6 +139,20 @@ def get_history(self, memory_id):
             for row in rows
         ]
 
+    def search_history(self, new_memory):
+        memory_list = [s.strip() for s in new_memory.split("- ") if s]
+
+        placeholders = ", ".join(["?"] * len(memory_list))
+        query = (
```

Comment:
can you please also check the user_id/session_id? Otherwise, you will be checking memories across other user_ids, which is inappropriate.